### PR TITLE
Add originInfo to top-level elements assessment

### PR DESCRIPTION
**Pull request by CanOfBees**
This PR adds originInfo examples to our top-level elements.

### CODE REVIEW

File: docs/source/top_level_elements/originInfo_dates.rst [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+About
+-----
+
+This section describes all of the different subelements of originInfo in our MODS.
+
+Dates
+-----
+
+We have three different date-related subelements: dateCreated, dateIssued, and dateOther.
```
Comment (mlhale7) [inline docs/source/top_level_elements/originInfo_dates.rst]: Technically we have four, right (with copyrightDate)? I know it's an edge case.

File: docs/source/top_level_elements/originInfo_dates.rst [hunk 2]
```diff
@@ -0,0 +1,282 @@
...
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> dcterms:created "approximately between 1940 and 1950", "1940~/1950" .
+
+dateIssued
+----------
+
+`dateIssued`'s attributes and possible values follow the examples in `dateCreated` very closely. The primary difference between the two is that `dateIssued` is used in records describing serials.
```
Comment (mlhale7) [inline docs/source/top_level_elements/originInfo_dates.rst]: Not super important, but dateIssued is broadly used to describe published materials (not just serials, though for us this is the vast majority) - https://www.loc.gov/standards/mods/userguide/origininfo.html#dateissued The sheet music collection is a good example of a non-serial collection that uses dateIssued.

File: docs/source/top_level_elements/originInfo_publishing.rst [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+      <dateCreated encoding="edtf" keyDate="yes">1665</dateCreated>
+    </originInfo>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators>
+    @prefix dcterms: <http://purl.org/dc/terms/>
+
+    <https://example.org/objects/1> relators:pbl "Archivision, Inc." ;
+        dcterms:created "1665/" .
```
Comment (mlhale7) [inline docs/source/top_level_elements/originInfo_publishing.rst]: Not sure how we're going to be able to handle this programmatically (translating "begun" to the start of a timespan), but I agree with the modeling of this information.

File: docs/source/top_level_elements/originInfo_dates.rst (hunk 1 above)
Comment (CanOfBees) [reply docs/source/top_level_elements/originInfo_dates.rst]: Thanks for catching this!

File: docs/source/top_level_elements/originInfo_dates.rst (hunk 2 above)
Comment (CanOfBees) [reply docs/source/top_level_elements/originInfo_dates.rst]: Thanks for catching this. I updated the descriptive text - let me know if it needs additional tweaking.

File: docs/source/top_level_elements/originInfo_publishing.rst (hunk 3 above)
Comment (CanOfBees) [reply docs/source/top_level_elements/originInfo_publishing.rst]: You're absolutely right: we're not! I updated the example to reflect our latest decisions on how to proceed.

**Review by mlhale7 — APPROVED** [review]
Thanks for these changes @CanOfBees !

**Pull request merged**